### PR TITLE
Aji13187/map pin not loaded

### PR DESCRIPTION
**Pull request by aji13187**
We were using an unsupported image format(SVG) for map pin in the Search view tool.

`PictureMarkerSymbol` supported image formats are BMP, GIF, ICO, JPEG, and PNG. not SVG.
https://developers.arcgis.com/qt/cpp/api-reference/esri-arcgisruntime-picturemarkersymbol.html#PictureMarkerSymbol-1

So converted the SVG image to PNG using GIMP tool on linux and used it and fixed the issue.

### CODE REVIEW

File: uitools/toolkitcpp/CMakeLists.txt [hunk 1]
```diff
@@ -102,6 +102,7 @@ set(esri_arcgisruntime_toolkit_images_resource_files
     "images/organization.svg"
     "images/pause.svg"
     "images/pin-tear.svg"
```
Comment (GuillaumeBelz) [inline uitools/toolkitcpp/CMakeLists.txt:104]: Maybe remove SVG unused files?

File: uitools/toolkitcpp/cpp/Esri/ArcGISRuntime/Toolkit/LocatorSearchSource.cpp [hunk 2]
```diff
@@ -36,7 +36,7 @@
 namespace Esri::ArcGISRuntime::Toolkit {
 
   namespace {
-    const char* MAP_PIN = "qrc:///esri.com/imports/Esri/ArcGISRuntime/Toolkit/images/pin-tear.svg";
+    const char* MAP_PIN = "qrc:///esri.com/imports/Esri/ArcGISRuntime/Toolkit/images/pin-tear.png";
```
Comment (GuillaumeBelz) [inline uitools/toolkitcpp/cpp/Esri/ArcGISRuntime/Toolkit/LocatorSearchSource.cpp:39]: constexpr?

**Review by GuillaumeBelz — APPROVED** [review]

File: uitools/toolkitcpp/CMakeLists.txt (hunk 1 above)
Comment (aji13187) [reply uitools/toolkitcpp/CMakeLists.txt:104]: `pin-tear.svg` is still used in callout demo to show the icon and it is working.

File: uitools/toolkitcpp/cpp/Esri/ArcGISRuntime/Toolkit/LocatorSearchSource.cpp (hunk 2 above)
Comment (aji13187) [reply uitools/toolkitcpp/cpp/Esri/ArcGISRuntime/Toolkit/LocatorSearchSource.cpp]: done.

**Review by JamesMBallard — APPROVED** [review]

**Comment by aji13187** [comment]
@JamesMBallard - Confirmed the other SVG images are not used where it is not supported. They are mainly for the icons on the toolkit.

**Pull request merged**